data/human/human missions.txt

```diff
@@ -5428,3 +5428,214 @@ mission "Deal Change 2"
+				goto taxi
+
+			label illness
+			action
+				set "Blind Man from Martini: label illness"
+				"Blind Man from Martini: taxi questions count" ++
+			`	"I don't know if I really should tell you this, but Anita was working on a project on nanomachines at CyberAcme when she got back. She was, uh... infected by them by accident. The experimental treatment is meant to try and eliminate the rogue machines, but since they're so small... that's why the treatment is experimental."`
+				goto taxi
+
+			label waiting
+			action 
+				"Blind Man from Martini: taxi questions count" ++
+			`	Deciding not to ask Douglas anything, you look out the windows of the taxi. You're in the midst of New Sydney, and skyscrapers surround you from each side of the avenue, but despite this it is still quite green and cheery, with trees on each side and people walking to work in the morning.`
+				to display
+					"Blind Man from Martini: taxi questions count" == 1
+				goto taxi
+
+			`	As the taxi drives on a highway on the side of a small hill, the window gives you a panorama of the suburbs of New Sydney and the two skylines on each side, one marking the commercial district where the spaceport is prominent in its gleaming steel-glass architecture, and the other being the University district, a series of clean and white, flowing buildings and skyscrapers.`
+				to display
+					"Blind Man from Martini: taxi questions count" == 2
+				goto taxi
+
+			`	The taxi enters the University district proper now, and the window shows thongs of university students from the University of New Sydney travelling to and from buildings, going over steel walkways and on footpaths beside the streets, clutching their backpacks and data pads. Your datapad shows that University isn't actually the only university district in New Sydney; the University of Chiron's campus is on the other side of town. Presumably, to keep the fierce competition between the two institutions from boiling over each other.`
+				to display
+					"Blind Man from Martini: taxi questions count" == 3
+				goto taxi
+
+			label hospital
+			`	Eventually, the taxi passes by a sign for the "UNS Health Sciences" quarter, which by the looks of it takes up an entire section of University district. The taxi pulls up beside the entrance of the university hospital, where you help Douglas out with his luggage. The building is both stark from the outside and strangely colorful on the inside, as if the person building it decided that hospitals were far too sterile and that people should be more easily treated in a place bright and alive than cold and white.`
```

Comment:
```suggestion
			`	Eventually, the taxi passes by a sign for the "UNS Health Sciences" quarter, which by the looks of it takes up an entire section of University district. The taxi pulls up beside the entrance of the university hospital, where you help Douglas out with his luggage. The building is both stark from the outside and strangely colorful on the inside, as if the person building it decided that hospitals were far too sterile and that people could be more easily treated in a place bright and alive than cold and white.`
```
or
```suggestion
			`	Eventually, the taxi passes by a sign for the "UNS Health Sciences" quarter, which by the looks of it takes up an entire section of University district. The taxi pulls up beside the entrance of the university hospital, where you help Douglas out with his luggage. The building is both stark from the outside and strangely colorful on the inside, as if the person building it decided that hospitals were far too sterile and that people should be treated in a place bright and alive instead of cold and white.`
```
or
```suggestion
			`	Eventually, the taxi passes by a sign for the "UNS Health Sciences" quarter, which by the looks of it takes up an entire section of University district. The taxi pulls up beside the entrance of the university hospital, where you help Douglas out with his luggage. The building is both stark from the outside and strangely colorful on the inside, as if the person building it decided that hospitals were far too sterile and that people should be treated in a place bright and alive instead of cold and white.`
```
All work better, i believe!